### PR TITLE
Two log color adjustments

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/basic/terminal-util.h [hunk 1]
```diff
@@ -21,7 +21,7 @@
 /* Bold/highlighted */
 #define ANSI_HIGHLIGHT_RED     "\x1B[0;1;31m"
 #define ANSI_HIGHLIGHT_GREEN   "\x1B[0;1;32m"
-#define ANSI_HIGHLIGHT_YELLOW  "\x1B[0;1;33m"
+#define ANSI_HIGHLIGHT_YELLOW  "\x1B[0;1;38;5;185m"
```
Comment (poettering) [inline src/basic/terminal-util.h:24]: hmm, this means non-highlight yellow is now very differently defined from highlight yellow...

I must say, I am a bit afraid of departing from palettable/themable color sets too much here.. It sucks that we did this for grey, but do we really want to do this for yellow too? I mean, grey is just greay, but yellow is a color!

Maybe we can rethink whether we need the color at all? or pick a different one?

File: src/shared/logs-show.c [hunk 2]
```diff
@@ -421,6 +429,9 @@ static int output_short(
         if (priority_len == 1 && *priority >= '0' && *priority <= '7')
                 p = *priority - '0';
 
+
+        audit = streq_ptr(transport, "audit");
```
Comment (poettering) [inline src/shared/logs-show.c:433]: not a major fan of such special coloring  of transport, but maybe it's ok...

File: src/basic/terminal-util.h (hunk 1 above)
Comment (keszybz) [reply src/basic/terminal-util.h:24]: I've been running journalctl with yellow locally (for me, there isn't much difference between the yellows), and I think it's a clear improvement in usability. It helps to distuinguish the warning logs from the rest.

Comment (keszybz) [reply src/basic/terminal-util.h:24]: I guess some kind of orange-y color would be OK too.

Comment (poettering) [reply src/basic/terminal-util.h:24]: is there orange in the palette colors?

Comment (poettering) [reply src/basic/terminal-util.h:24]: hmm, there appears not to be...

**Comment by tocomo** [comment]
Great. This kind of improvements is horrible. Now all users with bright terminal background colors are doomed.

**Pull request merged**